server/routes/api/variable.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from flask import Blueprint, request
+import services.datacommons as dc
+
+bp = Blueprint("variable", __name__, url_prefix='/api/variable')
+
+
+@bp.route('/path')
+def get_variable_path():
+    """Gets the path of a stat var to the root of the stat var hierarchy.
+    """
+    dcid = request.args.get("dcid")
```

Comment:
nit: call html.escape on the dcid before further processing? it should not affect valid dcid's

Reply:
This is directly passed to another url (instead of using as html text), so this should be fine?

Reply:
I think the problem is that the dcid is directly returned as part of the response

Reply:
yep that's the issue. and it gets `eval`ed by the browser so is a risk.

Reply:
Updated now